README.md

```diff
@@ -252,7 +320,10 @@ the file is automatically generated when leftwm or leftwm-check is run for the f
 | Mod + Shift + R     | Reload LeftWM and its config                                           |
 | Mod + p             | Use dmenu to start application                                         |
 
-**Note:** Although we encourage you to use [Alacritty](https://github.com/alacritty/alacritty), LeftWM will set your default terminal to the first terminal it finds in this list (in the order presented):
+**Note:** Although we encourage you to use [Alacritty](https://github.com/alacritty/alacritty),
+LeftWM will set your default terminal to the first terminal it finds in this list (in the order
+presented):
+
 1. Alacritty
```

Comment:
(Irrelevant to this PR, but I'm pretty sure this list has been changed. https://github.com/leftwm/leftwm/blob/1694b6a1feab688f45e2d89f71f54f46b75d9b88/src/config.rs#L95 ).

Reply:
Yup, it was. I updated it.